### PR TITLE
Add manage invites endpoints

### PR DESCRIPTION
**Pull request by PeterOcansey**
resolves #https://github.com/mailersend/mailersend/issues/6071

### Changelist

- [x] Add list invite endpoint
- [x] Add find invite endpoint
- [x] Add resend invite endpoint
- [x] Add cancel invite endpiont

### CODE REVIEW

File: src/Endpoints/Invite.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+class Invite extends AbstractEndpoint
+{
+    protected string $endpoint = 'invites';
+
+    /**
+     * @throws \Psr\Http\Client\ClientExceptionInterface
+     * @throws \JsonException
+     * @throws \MailerSend\Exceptions\MailerSendAssertException
+     */
+    public function getAll(): array
```
Comment (JoaoFSCruz) [inline src/Endpoints/Invite.php:17]: This should have pagination.

**Review by JoaoFSCruz — CHANGES_REQUESTED** [review]

**Review by JoaoFSCruz — APPROVED** [review]

**Comment by JoaoFSCruz** [comment]
@PeterOcansey I think we can release this, right?

**Comment by PeterOcansey** [comment]
@JoaoFSCruz I missed out README, I just updated it now....let me know if ok to release

File: README.md [hunk 2]
```diff
@@ -127,6 +127,10 @@ MailerSend PHP SDK
         * [Add a User](#add-a-user)
         * [Update a User](#update-a-user)
         * [Delete a User](#delete-a-user)
+        * [Get a list of Invites](#get-a-list-of-invites)
+        * [Get a single Invite](#get-a-single-invite)
+        * [Resend an Invite](#resend-an-invite)
+        * [Cancel and Invite](#cancel-an-invite)
```
Comment (JoaoFSCruz) [inline README.md:133]:
```suggestion
        * [Cancel an Invite](#cancel-an-invite)
```

**Review by JoaoFSCruz — APPROVED** [review]
My bad, also forgot about the README. Just a typo and we're good.

**Pull request merged**